src/openrct2-ui/windows/Ride.cpp

```diff
@@ -3567,10 +3567,26 @@ namespace OpenRCT2::Ui::Windows
 
                 ft.Rewind();
                 ft.Increment(10);
-                ft.Add<StringId>(STR_FORMAT_SECONDS);
-                ft.Add<uint16_t>(ride->minWaitingTime);
-                ft.Add<StringId>(STR_FORMAT_SECONDS);
-                ft.Add<uint16_t>(ride->maxWaitingTime);
+                int minWait = ride->minWaitingTime;
+                int maxWait = ride->maxWaitingTime;
+
+                ft.Rewind();
+                ft.Increment(10);
+
+                if (minWait == 1) {
+                    ft.Add<StringId>(STR_FORMAT_SECOND);
+                }
+                else {
+                    ft.Add<StringId>(STR_FORMAT_SECONDS);
+                }
```

Comment:
This does not match our code guidelines. See `clang-format` CI for suggestions.

Also consider making this a ternary, e.g.:
```suggestion
                ft.Add<StringId>(minWait == 1 ? STR_FORMAT_SECOND : STR_FORMAT_SECONDS);
```